operations/preinstall/Change_River_BMC_Credentials.md

```diff
@@ -0,0 +1,136 @@
+## Fresh Install: Setting NodeBMC and RouterBMC Redfish Credentials
+
+These steps are performed before the installation of Shasta System Management
+or HPCM management software stacks.  The goal is to set the BMC Redfish 
+credentials to values that the management software will be expecting so that
+all software systems work smoothly with the Redfish hardware.
+
+## Prerequisites
+
+Before doing these operations, the following is assumed:
+
+- There is a workstation or laptop which can access all target BMCs
```

Comment:
Super nit: there's no period here, but there is after the bullets.  I could go either way on whether periods are actually required here, but they should probably be consistent.

Reply:
Fixed.